chart/values.schema.json

```diff
@@ -4588,6 +4588,56 @@
                                 "verify-full"
                             ],
                             "default": "disable"
+                        },
+                        "livenessProbe": {
+                            "description": "LivenessProbe configurations for ``metricsExporterSidecar``",
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "initialDelaySeconds": {
+                                    "description": "Metrics Exporter liveness probe initial delay",
+                                    "format": "int32",
+                                    "type": "integer",
+                                    "default": 10
+                                },
+                                "periodSeconds": {
+                                    "description": "Metrics Exporter liveness probe frequency",
+                                    "format": "int32",
```

Comment:
Does helm honor these formats? We don't have them for anything else, and the one instance of it for periodSeconds in the upstream k8s schema has this as int64.

I'd be inclined to just remove all the formats you are adding here, unless you are sure helm actually checks them.

Reply:
I reviewed it again and pretty agree that it's quite extra to be here. Removed format fields, thank you!